xml/System.Windows/TextMarkerStyle.xml

```diff
@@ -143,7 +143,7 @@
       </ReturnValue>
       <MemberValue>5</MemberValue>
       <Docs>
-        <summary>A lowercase Roman numeral is displayed, starting with the numeral i, for example, i, ii, iii, and iv. The <see cref="T:System.Windows.TextMarkerStyle" /> is automatically incremented for each item added to the list.</summary>
+        <summary>A lowercase Roman numeral starting with the numeral i. For example, i, ii, iii, and iv. The numeral is automatically incremented for each item added to the list.</summary>
```

Comment:
you added is displayed to some summaries and removed from others. By design?

Reply:
I'd meant to remove all occurrences of "is displayed." Bugt I don't see any  remaining, @mairaw.

Reply:
Duh. My brain somehow was seeing some changes in reverse. Yes, you're right @rpetrusha.

Reply:
That's OK -- I've done it myself more than once, @mairaw.